slycot/setup.py

```diff
@@ -21,10 +21,17 @@ def configuration(parent_package='', top_path=None):
 
     f2py_sources = ['src/_wrapper.pyf']
 
+    pyver = sysconfig.get_config_var('VERSION')
+
     if sys.platform == 'win32':
         liblist = ['liblapack', 'libblas']
     else:
-        liblist = ['lapack']
+        # this is needed on Py 3.x, and fails on Py 2.7
+        try:
+            abiflags = sys.abiflags
+        except AttributeError:
+            abiflags = ''
+        liblist = ['lapack', 'python'+pyver+abiflags]
```

Comment:
Linking on Mac OSX failed; when linking a shared library on mac OSX (and on some strict Linux platforms as well, SUSE is one of the stricter ones), the presence of all symbols that the library in turn requires is checked. Only lapack was in the list, but for the python binding also libpython is required. The alternative is relaxing the linking with flags, but this is much better, you at least also know that all symbols are present, and not figure that out when you try to load the module.